src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -411,6 +412,19 @@ private function getRandomValueForProductAttribute(ProductAttributeInterface $pr
             case ProductAttributeValueInterface::STORAGE_DATE:
             case ProductAttributeValueInterface::STORAGE_DATETIME:
                 return $this->faker->dateTimeThisCentury;
+            case ProductAttributeValueInterface::STORAGE_JSON:
+                switch ($productAttribute->getType()) {
```

Comment:
Can't we do an `if` instead of switch? `if ($productAttribute->getType() !== SelectAttributeType::TYPE) { throw ... }`